components/mpas-albany-landice/src/Registry.xml

```diff
@@ -1243,6 +1243,12 @@ is the value of that variable from the *previous* time level!
                 <var name="groundedSfcMassBalApplied" type="real" dimensions="nCells Time" units="kg m^{-2} s^{-1}"
                      description="applied surface mass balance on grounded ice"
                 />
+                <var name="bareIceAblation" type="real" dimensions="nCells Time" units="kg m^{-2} s^{-1}"
```

Comment:
As noted elsewhere, I'm not sure we need `bareIceAblation`, since I think we will only ever use `bareIceAblationApplied` for time averaging.

Reply:
Holding on to this field until we decide whether or not it is needed for estimating energy conservation errors.